runtime/include/zmq_helper.h

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
I don't think this file belongs in `runtime` at all. Rather, I think it belongs in `modules/packages`, e.g. `modules/packages/ZmqHelper/`. See `modules/packages/CryptoHandlers/` for a precedent in this area.